### PR TITLE
Add initial emacs keybindings and README instructions

### PR DESCRIPTION
**Pull request by charlespierce**
Info
-----
* Create the README with installation / uninstallation instructions
* Add the initial Emacs keybindings file (it's relatively short because most of our keybindings already use emacs shortcuts)

Notes
-----
* There are still a number of actions that we don't properly support in Warp, which we will need to update before we can create the appropriate key bindings in this file.

### CODE REVIEW

**Comment by gagata** [comment]
> it's relatively short because most of our keybindings already use emacs shortcuts

regarding this - does it mean it relies on the default shortcuts (for other actions) being used? are those also editable?

i'm kinda wondering if it would make sense to keep the structure of those preset files so that it's easy to "combine" them all together? say, you have a preset for navigating the blocks that uses some set of preferred shortcuts, but a different kind of preset for the editor bindings. maybe we could just add comments and separate "sections" in files for readability?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+
+### Disclaimer
+
+These presets are a work in progress and may be missing some functionality. As we continue to improve Warp and support more behaviors, they will be updated. If you have any questions or run into any issues, please reach out to us either on [Discord](https://discord.gg/warpdotdev) or in our [Issues Repository](https://github.com/warpdotdev/Warp)
+
+## Installation
+
+To use any of these keyboard presets, follow these steps:
+
+1. Confirm that you have a `~/.warp` directory. If you don't, create it with `mkdir ~/.warp`
```
Comment (zachlloyd) [inline README.md:13]: unrelated to this PR, but this would be great as a workflow @ianhodge @alokedesai

File: emacs.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (zachlloyd) [inline emacs.yaml:1]: are we publishing the file format for this somewhere?  i.e. what are all of the available bindings?

seems like it should be in this repo or at least linked to

File: README.md [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
+2. If you have an existing `~/.warp/keybindings.yaml` file, back it up by renaming it to `keybindings.yaml.bak`
+3. Copy the appropriate `.yaml` file from this repo into `~/.warp/`
+4. Rename that file to `keybindings.yaml`
+5. Restart Warp
+
+That's it! When Warp re-opens it will pick up your new keyboard shortcuts and automatically apply them.
```
Comment (zachlloyd) [inline README.md]: maybe make a note on what happens if you subsequently edit the bindings through the ui?

**Review by zachlloyd — APPROVED** [review]

**Comment by zachlloyd** [comment]
i had this same thought and like the proposed solution as a stopgap

File: README.md (hunk 3 above)
Comment (charlespierce) [reply README.md]: I like that, will do!

File: emacs.yaml (hunk 2 above)
Comment (charlespierce) [reply emacs.yaml:1]: We can, however I'm not sure of the best way to present it. The available actions are tied to our internal implementation and will continue to grow as we add new actions. Maintaining a full list of available actions will add some ongoing maintenance burden.

**Comment by zachlloyd** [comment]
if we don't publish the bindings i think it doesn't make a lot of sense to
have this as an open repo - let's maybe discuss quickly?

File: emacs.yaml (hunk 2 above)
Comment (elviskahoro) [reply emacs.yaml:1]: Maybe, we can just loop this under documentation and I can manage keeping it up to date.

Comment (charlespierce) [reply emacs.yaml:1]: Updated to include a `FORMAT.md` file that describes the file format and lists the available actions.

File: FORMAT.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
+# Keybindings File Format
```
Comment (zachlloyd) [inline FORMAT.md:1]: this is great.  we might want to mention that this format is subject to change

**Pull request merged**